docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -59,6 +63,8 @@ For a detailed walk-through of the runtime flow, see the [Internal design flow](
 ---
 ### Prerequisites
 
+* Configuration overrides can only be used in combination with Domain in Image and Domain in PV domains (the `domainHomeSourceType` must be either `PersistentVolume` or `Image`). For Model in Image domains (`domainHomeSourceType` is `FromModel`), use [Model in Image Runtime Updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}) instead.
+
```

Comment:
A similar statement exists in the beginning of this doc as a notice (see Overview section above). It is okay if the repetition is intentional and we would like to emphasize this.

Reply:
Yah. I did that on purpose.